contracts/Aavegotchi/init/InitItemsRolesRegistryFacet.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.1;
+
+import {LibDiamond} from "../../shared/libraries/LibDiamond.sol";
+import {IERC1155Receiver} from "@openzeppelin/contracts/token/ERC1155/IERC1155Receiver.sol";
+import {IERC7589} from "../../shared/interfaces/IERC7589.sol";
+
+contract InitItemsRolesRegistryFacet {
+    function init() public {

```

Comment:
since this function is to be called once i.e during deployment, the function selector can be removed immediately after being added and executed via the upgrade

Reply:
this deletes the entire `initItemsRolesRegistryFacet` facet

Reply:
Fixed!